Minor updates to Remote Access (#3622)
raspberrypi/documentation: documentation/asciidoc/computers/remote-access/raspberry-pi-connect.adoc

[[raspberry-pi-connect]]
== Screen share with Raspberry Pi Connect

> You can access the desktop of a Raspberry Pi remotely from another device using Raspberry Pi Connect. Raspberry Pi Connect shares your screen in a browser window and handles configuration for you, so you don't have to find your Raspberry Pi's local IP address or modify your local network.

Author: ```suggestion
You can access the desktop of a Raspberry Pi remotely from a browser on another device using Raspberry Pi Connect. Raspberry Pi Connect handles configuration automatically, so you don't have to find your Raspberry Pi's local IP address or modify your local network.
```
